gating/check/run

```diff
@@ -187,8 +187,8 @@ bash scripts/bootstrap-ansible.sh
 _download_dependent_components
 
 if [ "${RE_JOB_SCENARIO}" = "ansible_lint" ]; then
+   find ./playbooks -type f -regex '.*\.y[a]?ml' -print0 | xargs -0 ${LINT_COMMAND} -x $LINT_SKIPLIST --exclude=/etc
    exit 0
```

Comment:
We should drop the `exit 0` if it is no longer needed since the lint is passing.